capture_service/device_mgr.h

```diff
@@ -92,15 +96,17 @@ class AndroidDevice
     AdbSession       &Adb() { return m_adb; }
 
     AndroidApplication *GetCurrentApplication() { return app.get(); }
-    absl::Status        RetrieveTraceFile(const std::string &trace_file_path,
-                                          const std::string &save_path);
+    absl::Status        RetrieveTrace(const std::string &trace_file_path,
+                                      const std::string &save_path,
+                                      const bool         isGfxrCapture);
 
 private:
     const std::string                   m_serial;
     DeviceInfo                          m_dev_info;
     AdbSession                          m_adb;
     DeviceState                         m_original_state;
     std::unique_ptr<AndroidApplication> app;
```

Comment:
not really your code, but could you update this member to have prefix `m_` to be consistent with other members? thanks :)